src/django/api/management/commands/makefixtures.py

```diff
@@ -182,14 +187,16 @@ def make_facility_lists(max_id=14):
 
 def make_facility_list_item(list_pk, item_pk, row_index, raw_data):
     (created_at, updated_at) = make_created_updated()
+    status = random.choice([s for (s, _) in FacilityListItem.STATUS_CHOICES])
```

Comment:
We were running into problems testing our batch processing in #195 because the fixture data was inconsistent with the way data will actually appear in the database during normal usage.

Any random choices we make when building fixtures need to be threaded through the FacilityListItem->FacilityMatch->Facility relationship correctly.

Here is the commit where I fixed this as part of #195

https://github.com/open-apparel-registry/open-apparel-registry/pull/195/commits/1ad3ede81c867094f50cd5abee75413d66c05b4b

Reply:
Cool. Should that make it so there is some match data in the existing set of fixtures to render? If so, I can adjust the changes for this second commit.

Reply:
I think you will need to extend the idea. The version in #195 results in only having `UPLOADED` and `MATCHED` list items. To get some things in the in between states, we will just have to make sure that the `status` of the `FacilityListItem` is set to `POTENTIAL_MATCH` when there is a related `FacilityMatch`.

Reply:
Got it, that makes sense. I'll adapt it to work with this.